README.markdown

```diff
@@ -19,28 +20,22 @@ as depicted in this diagram:
 ### Installation
 
 _spray-json_ is available from the [repo.spray.io] repository.
-The latest release is `1.2.6` and is built against Scala 2.9.3, Scala 2.10.4 and Scala 2.11.0-RC4.
+The latest release is `1.3.0` and is built against Scala 2.10.4 and Scala 2.11.2.
```

Comment:
Let's remove patch versions as they shouldn't matter.

Reply:
I changed that in https://github.com/spray/spray-json/pull/239